src/Pacman/Player/index.js

```diff
@@ -76,21 +76,25 @@ export default class Player extends Component {
         }
 
         if (this.props.onEnd) {
-            setImmediate(() => this.props.onEnd());
+            if(this.props.won) {
+                setImmediate(() => this.props.onEnd("WON"));
+            } else {
+                setImmediate(() => this.props.onEnd("LOST"));
+            }
         }
 
         return null;
     }
     componentDidUpdate(prevProps) {
-        if (!prevProps.lost && this.props.lost) {
+        if ((!prevProps.lost && this.props.lost) || (!prevProps.won && this.props.won)) {
```

Comment:
Instead of having `lost` and `won`, a better approach would be to have a single `ended` state, which would be set to a constant `WON` or `LOST` (where they are strings).